### PR TITLE
UNI-37: implement sign out button

### PR DESCRIPTION
**Pull request by vivianchengg**

### CODE REVIEW

File: frontend/src/pages/Chats.tsx [hunk 1]
```diff
@@ -1,5 +1,21 @@
-import { Box } from '@mui/material';
+import { Box, AppBar, Toolbar } from '@mui/material';
+import { SignOutButton } from '../components/SignOutButton';
 
 export const Chats = () => {
-  return <Box>This is the Chats Page!</Box>;
+  return (
+    <div>
+      <ButtonAppBar />
```
Comment (Siyer2) [inline frontend/src/pages/Chats.tsx]: This TopAppBar component will be used across many different pages e.g. imagine if we add a page to update a user's faculty. So what we need to do is move the ButtonAppBar to its own file in the components folder.

File: frontend/src/pages/Chats.tsx [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
 };
+
+export default function ButtonAppBar() {
+  return (
+    <AppBar position="static">
+      <Toolbar>
+        <Box display="flex" flexGrow={1}></Box>
+        <SignOutButton />
+      </Toolbar>
+    </AppBar>
```
Comment (Siyer2) [inline frontend/src/pages/Chats.tsx]: When we move this to the components folder, we still need a way for the chats page to display all the chat information below the AppBar. This is how we can do it - we pass the chat information as a 'children' prop to the AppBar. Our ultimate goal is that the chats page looks something like this:
```
export const Chats = () => {
  return (
    <TopAppBar>
      <div>chat messages here etc etc</div>
    </TopAppBar>
  );
};
```

HINT: in the new TopAppBar component, you'll need to be able to pass children elements into it. 
HINT2: Check out how I've done it in the [PageLayout component](https://github.com/Siyer2/uni-connector/blob/73a654e148228e1595d7c1c64cfe4d853136406c/frontend/src/components/PageLayout.tsx#L17).

**Review by Siyer2 — CHANGES_REQUESTED** [review]
Almost there with it Vivian!

File: frontend/src/pages/Chats.tsx (hunk 2 above)
Comment (vivianchengg) [reply frontend/src/pages/Chats.tsx]: got it! Please review changes!

**Review by Siyer2 — APPROVED** [review]
Nice!

**Pull request merged**